### PR TITLE
Onboarding + NavBar

### PR DESCRIPTION
**Pull request by adamG0315**
Onboarding UI changes + Bottom Nav Bar Potential fix: Sticky at the bottom.

### CODE REVIEW

File: ff.web/src/components/organisms/BottomNavigation.tsx [hunk 1]
```diff
@@ -61,12 +61,12 @@ const BottomNavigation = () => {
     };
   
     if(isVisible) return (
-      <div>
-          <TabBar activeKey={pathname} onChange={handleTabChange}>
-            {tabs().map(item => (
-              <TabBar.Item key={item.key} icon={item.icon} title={item.title} />
-            ))}
-          </TabBar>
+      <div style={{ position: 'sticky', bottom: 0, left: 0, right: 0, zIndex: 1000 }}>
```
Comment (adamG0315) [inline ff.web/src/components/organisms/BottomNavigation.tsx:64]: No need to position TabNavigation this way. For mobile layout flex is a better alternative. Remove this, it is fixed in my branch, which I am going to push soon.

File: ff.web/src/scss/components/_onboarding.scss [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+    display: flex;
+    flex-direction: column;
+    height: 100vh;
+    width: 100%;
+    background-color: var(--surface-primary);
+    color: var(--text-primary);
+  }
+
+.onboarding-header {
+  padding: 10px;
```
Comment (adamG0315) [inline ff.web/src/scss/components/_onboarding.scss:11]: use spacing variable from the design system

File: ff.web/src/scss/components/_onboarding.scss [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+  position: relative;
+}
+
+.onboarding-content {
+    flex: 1;
+    overflow-y: auto;
+    padding: 20px;
+    padding-bottom: 120px; // Adjust based on your navigation height
```
Comment (adamG0315) [inline ff.web/src/scss/components/_onboarding.scss:19]: we have spacing variables for this

File: ff.web/src/scss/components/_onboarding.scss [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+  }
+
+.onboarding-step {
+  max-width: 600px;
+  width: 100%;
+}
+
+.onboarding-navigation {
+    position: fixed;
+    bottom: 56px; // Adjust this value based on your app's bottom navigation height
```
Comment (adamG0315) [inline ff.web/src/scss/components/_onboarding.scss:29]: and for this

File: ff.web/src/scss/components/_onboarding.scss [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+    left: 0;
+    right: 0;
+    background-color: var(--surface-secondary);
+    padding: 16px;
```
Comment (adamG0315) [inline ff.web/src/scss/components/_onboarding.scss:33]: and for this too

File: ff.web/src/scss/components/_onboarding.scss [hunk 6]
```diff
@@ -0,0 +1,212 @@
...
+    z-index: 1000;
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
```
Comment (adamG0315) [inline ff.web/src/scss/components/_onboarding.scss:37]: same here

File: ff.web/src/scss/components/_onboarding.scss [hunk 7]
```diff
@@ -0,0 +1,212 @@
...
+  }
+
+h2 {
+  font-size: 2.5rem;
```
Comment (adamG0315) [inline ff.web/src/scss/components/_onboarding.scss:41]: if the font size in the typography variables doesn't give the right result, try a different h tag, as they are styled already

File: ff.web/src/scss/components/_onboarding.scss [hunk 8]
```diff
@@ -0,0 +1,212 @@
...
+  margin-bottom: 1rem;
+}
+
+p {
+  font-size: 1.2rem;
```
Comment (adamG0315) [inline ff.web/src/scss/components/_onboarding.scss:46]: same here

File: ff.web/src/scss/components/_onboarding.scss [hunk 9]
```diff
@@ -0,0 +1,212 @@
...
+  margin-bottom: 2rem;
+}
+
+.onboarding-option {
+  display: flex;
+  align-items: center;
+  margin-bottom: 1rem;
```
Comment (adamG0315) [inline ff.web/src/scss/components/_onboarding.scss:53]: we have values for these too

File: ff.web/src/scss/components/_onboarding.scss [hunk 10]
```diff
@@ -0,0 +1,212 @@
...
+}
+
+.onboarding-radio {
+  display: none;
+}
+
+.onboarding-label {
+  display: flex;
+  flex-direction: column;
+  margin-left: 1rem;
```
Comment (adamG0315) [inline ff.web/src/scss/components/_onboarding.scss:72]: use spacing variable

File: ff.web/src/scss/components/_onboarding.scss [hunk 11]
```diff
@@ -0,0 +1,212 @@
...
+}
+
+.onboarding-label-title {
```
Comment (adamG0315) [inline ff.web/src/scss/components/_onboarding.scss:75]: it is better to choose from an already exisiting and styled tag (h1-h4) or p

File: ff.web/src/scss/components/_onboarding.scss [hunk 12]
```diff
@@ -0,0 +1,212 @@
...
+  font-size: 1.2rem;
+  font-weight: bold;
+}
+
+.onboarding-label-description {
+  font-size: 0.9rem;
```
Comment (adamG0315) [inline ff.web/src/scss/components/_onboarding.scss:81]: too many custom font-sizes

File: ff.web/src/scss/components/_onboarding.scss [hunk 13]
```diff
@@ -0,0 +1,212 @@
...
+  color: #aaa;
+}
+
+.onboarding-button {
+  display: inline-block;
+  margin: 0.5rem;
+  padding: 0.5rem 1rem;
+  border: none;
+  border-radius: 20px;
```
Comment (adamG0315) [inline ff.web/src/scss/components/_onboarding.scss:90]: and border radius too

File: ff.web/src/scss/components/_onboarding.scss [hunk 14]
```diff
@@ -0,0 +1,212 @@
...
+  transition: background-color 0.3s ease;
+
+  &:hover, &.selected {
+    background-color: #4a4a4a;
+  }
+}
+
+.onboarding-number-input {
+  width: 100%;
+  padding: 0.5rem;
```
Comment (adamG0315) [inline ff.web/src/scss/components/_onboarding.scss:103]: var

File: ff.web/src/scss/components/_onboarding.scss [hunk 15]
```diff
@@ -0,0 +1,212 @@
...
+  font-size: 1.2rem;
+  background-color: #2a2a2a;
+  border: 1px solid #3a3a3a;
+  border-radius: 4px;
```
Comment (adamG0315) [inline ff.web/src/scss/components/_onboarding.scss:107]: var

File: ff.web/src/scss/components/_onboarding.scss [hunk 16]
```diff
@@ -0,0 +1,212 @@
...
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 1rem;
+}
+
+.onboarding-note {
+  font-size: 0.9rem;
+  color: #aaa;
+  margin-top: 1rem;
```
Comment (adamG0315) [inline ff.web/src/scss/components/_onboarding.scss:121]: var

File: ff.web/src/scss/components/_onboarding.scss [hunk 17]
```diff
@@ -0,0 +1,212 @@
...
+}
+
+.onboarding-search-input {
+  width: 100%;
+  padding: 0.5rem;
```
Comment (adamG0315) [inline ff.web/src/scss/components/_onboarding.scss:126]: var

File: ff.web/src/scss/components/_onboarding.scss [hunk 18]
```diff
@@ -0,0 +1,212 @@
...
+  font-size: 1.2rem;
+  background-color: #2a2a2a;
+  border: 1px solid #3a3a3a;
+  border-radius: 4px;
```
Comment (adamG0315) [inline ff.web/src/scss/components/_onboarding.scss:130]: var

File: ff.web/src/scss/components/_onboarding.scss [hunk 19]
```diff
@@ -0,0 +1,212 @@
...
+  color: #ffffff;
+  margin-bottom: 1rem;
+}
+
+.onboarding-selected-items {
+  display: flex;
+  flex-wrap: wrap;
+  margin-bottom: 1rem;
```
Comment (adamG0315) [inline ff.web/src/scss/components/_onboarding.scss:138]: var

File: ff.web/src/scss/components/_onboarding.scss [hunk 20]
```diff
@@ -0,0 +1,212 @@
...
+}
+
+.onboarding-selected-item {
+  margin: 0.25rem;
```
Comment (adamG0315) [inline ff.web/src/scss/components/_onboarding.scss:142]: var

File: ff.web/src/scss/components/_onboarding.scss (hunk 2 above)
Comment (RobertoHanas) [reply ff.web/src/scss/components/_onboarding.scss:11]: did you review the latest PR or just the first one?

**Comment by adamG0315** [comment]
trying to reopen

**Comment by adamG0315** [comment]
asdf

**Pull request closed without merging**